setup.cfg

```diff
@@ -76,4 +76,15 @@ column_limit = 120
 
 [options.entry_points]
 console_scripts =
-  run_topostats = topostats.run_topostats:main
\ No newline at end of file
+  run_topostats = topostats.run_topostats:main
+# See the docstring in versioneer.py for instructions. Note that you must
```

Comment:
Stuff below can be deleted (as it's in `pyproject.toml`)?

Reply:
Thanks I'd written it into `pyproject.toml` but didn't check `setup.cfg` and it could be `versioneer` added these commented lines.